clients/benchmarks/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_include_directories( client
     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
 )
 
+link_libraries( ${BOOST_ROOT}/lib )
```

Comment:
`/home/tim/usr/ **cget** /pkg/kitware__cmake/install/bin/cmake`

Are you using Paul's cget?